etc/builtin.conf

```diff
@@ -46,18 +46,26 @@ vlang=
 alang=
 slang=
 
-[gpu-hq]
-scale=spline36
-cscale=spline36
-dscale=mitchell
-dither-depth=auto
+[fast]
+scale=bilinear
+dscale=bilinear
+dither=no
+correct-downscaling=no
+linear-downscaling=no
+sigmoid-upscaling=no
+hdr-compute-peak=no
+
+[high-quality]
+scale=ewa_lanczossharp
 hdr-peak-percentile=99.995
 hdr-contrast-recovery=0.30
 allow-delayed-peak-detect=no
-correct-downscaling=yes
-linear-downscaling=yes
-sigmoid-upscaling=yes
 deband=yes
+scaler-lut-size=8
```

Comment:
this is new. does it have a relevant effect on quality?

Reply:
According to documentation it does. 

> All weights are linearly interpolated from those samples, so increasing the size of lookup table might improve the accuracy of scaler.

And it doesn't cost almost anything, changing lut size from 64x2 to 256x2 for HQ profile is imho good.

Reply:
"turning up number can improve quality" so much is obvious

> And it doesn't cost almost anything

that begs the question why is it not enabled by default?

What I'm saying here is that the combination of "improves quality", "costs nothing" and "is not default" sounds suspiciously like a placebo setting. I don't have a technical reason to present.

Reply:
In libplacebo we have a marginal use case for hard coding scaler LUTs in the shader body, the cutoff for this is 64 samples. Of course it doesn't matter for mpv.

I'm pretty sure I set this value by benchmarking on my GTX 970 several years ago. Caches have gotten bigger since then. Much bigger.